src/javascripts/test/e2e/filterViewSpec.js

```diff
@@ -165,7 +165,7 @@ describe('List filter', function () {
             $$('.filters .filter:nth-child(1) input').sendKeys('be');
             browser.driver.sleep(600); // debounce delay
             $$('ma-datagrid-pagination .total').then(function (totalElements) {
-                expect(totalElements[0].getText()).toBe('1 - 5 on 5');
+                expect(totalElements[0].getText()).toBe('1 - 4 on 4');
```

Comment:
OK, FakeRest doesn't handle the q search inside composite fields (the 5th comment has an author.name containing 'be'.